libs/server-sdk/src/data_sources/polling_data_source.cpp

```diff
@@ -0,0 +1,207 @@
+#include <boost/json.hpp>
```

Comment:
I've reduced duplication in lots of other places, but not here. Not confident in the value as of yet, but refactoring it would be internal.